proto/rill/admin/v1/api.proto

```diff
@@ -1977,6 +1984,9 @@ message GetReportMetaRequest {
   string owner_id = 5;
   google.protobuf.Timestamp execution_time = 6;
   repeated string email_recipients = 7;
+  bool anon_recipients = 8;
+  repeated ResourceName resources = 9;
+  ReportOptions.OpenMode web_open_mode = 10;
```

Comment:
Try to avoid nested types especially when they are used outside the nested scope itself (we haven't always been consistent with that, but try to going forward). 

So based on the previous, if you decide to go with the Go-defined enum, this can just be a `string` type. Otherwise, I suggest moving this to the outer scope as a `ReportOpenMode` enum or something like that.